packages/api-queries/src/site-scan.ts

```diff
@@ -47,6 +55,15 @@ export const unignoreThreatMutation = ( siteId: number ) =>
 		},
 	} );
 
+export const bulkFixThreatsMutation = ( siteId: number ) =>
```

Comment:
Same here, could it be `fixThreatsMutation`

Reply:
Changed.